ajax/libs/multi-level-push-menu/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "multi-level-push-menu",
+  "filename": "jquery.multilevelpushmenu.min.js",
+  "author": "adgsm",
```

Comment:
Let's use `Momčilo Džunić <it@make.rs>`
Refer https://github.com/adgsm/multi-level-push-menu/blob/master/bower.json#L5-L7.